error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/util/ThirdPartyLibrary.java

```diff
@@ -90,7 +90,9 @@ private static boolean canLoadClass(String className, VisitorState state) {
     try {
       classFinder.loadClass(state.getSymtab().unnamedModule, binaryName);
       return true;
-    } catch (CompletionFailure e) {
+    } catch (
+        @SuppressWarnings("java:S1166" /* Not exceptional. */)
```

Comment:
```suggestion
        @SuppressWarnings("java:S1166" /* Exception not relevant. */)
```
Perhaps? Or "Exception expected."

Reply:
Hmm, the exception is relevant, but I wouldn't say it's "expected". It's just "not unexpected", i.e.not exceptional.